### PR TITLE
Image show headers ui changes and pw tests to check functionality

### PR DESCRIPTION
**Pull request by mudhoney**
- Change some wordings 
- Add roles to be nice with playwright
- implement tests for showing image layer headers dialog box functionality

### CODE REVIEW

File: tests/page_objects/image_layer.ts [hunk 1]
```diff
@@ -195,6 +197,23 @@ class ImageLayer {
...
+     */
+    async showImageHeader(): Promise<void> {
+        await this.layer_controls.locator(".image-info-dialog-btn").click();
+    }
+
+    /**
+     * Parse image header dialog box for assertions.
+     * @return {ImageHeaderDialog} , parsed image header dialog box
+     */
+    async getImageHeaderDialog(): ImageHeaderDialog {
```
Comment (dgarciabriseno) [inline tests/page_objects/image_layer.ts]:
```suggestion
    async getImageHeaderDialog(): Promise<ImageHeaderDialog> {
```

File: tests/page_objects/image_headers_dialog.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+     * @param {string}        selected , selected tab name
+     * @return void
+     */
+    async assertTabs(tags: Array<string>, selected: string): Promise<void>{
+
+        for(let t of tags) {
+            await expect(this.box.locator('.image-info-dialog-menu').getByText(t)).toBeVisible()
+        }
+
+        await expect(this.box.locator('.image-info-dialog-menu').getByText("["+selected+"]")).toBeVisible()
```
Comment (dgarciabriseno) [inline tests/page_objects/image_headers_dialog.ts:44]:
```suggestion
        await expect(await this.box.locator('.image-info-dialog-menu').getByText("["+selected+"]")).toBeVisible()
```

File: tests/page_objects/image_headers_dialog.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            const headerTag = await this.box.locator('visible=true').getByRole('listitem').nth(index);
+            await expect(headerTag).toHaveText(imageHeaders[index].join(': '));
+        }
+    }
+
+    /**
+     * Trigger sorting of the shown headers.
+     * @return void
+     */
+    async sortHeaders(): Promise<void>{
```
Comment (dgarciabriseno) [inline tests/page_objects/image_headers_dialog.ts]: should this be `toggleSort`?
Does clicking it again "unsort"

File: tests/page_objects/image_headers_dialog.ts (hunk 2 above)
Comment (dgarciabriseno) [reply tests/page_objects/image_headers_dialog.ts:44]: this doesn't need await, my mistake.

File: tests/page_objects/image_headers_dialog.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        await this.box.locator('.image-info-sort-btn').click()
+    }
+
+    /**
+     * Trigger switch to tab.
+     * @param {strong} tab, tabname to swich to
```
Comment (mudhoney) [inline tests/page_objects/image_headers_dialog.ts]: string

**Review by dgarciabriseno — APPROVED** [review]

**Review by dgarciabriseno — APPROVED** [review]

**Pull request merged**